classes/woocommerce-schema.php

```diff
@@ -503,7 +503,7 @@ protected function add_individual_offers( $product ) {
 				],
 			];
 
-			$data[] = $offer;
+            		$data[] = apply_filters('wpseo_schema_offers', $offer, $variation);
```

Comment:
Hi @Dekadinious!

Thank you very much for the pull!

There's three minor improvements I'd love to see:
- Our filters generally use the singular form ( see `wpseo_schema_product` and `wpseo_schema_review` above ).
- I believe also passing the `$product` as an additional parameter would improve the overall usability of the filter.
- The filter is currently lacking documentation, you can see the [WordPress documentation standards for filters here](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#4-hooks-actions-and-filters).

```suggestion
            		$data[] = apply_filters( 'wpseo_schema_offer', $offer, $variation, $product );
```